tests/house_tests.py

```diff
@@ -25,6 +27,7 @@ class TestFilterHouseForm:
     def test_filter_parent_profession_1(self):
         pass_filter_house_create()
         HouseFactory(
+            user=UserFactory(),
```

Comment:
Why did you choose to use here UserFactoty() and on line 40 there is a test which gets new_user as an argument?
Are they the same? or is there a difference in the meaning? 
Is there a difference in how you run these tests? 
From what I understood, UserFactory generates a new User with random username & password, 
But on line 40, when you provide new_user as an argument, when you run this test you need to provide it with an existing User. Right?

Would't it be better if all of the test would use UserFactory? (If I understood correctly of course)

Reply:
you right, 
I will use only UserFactory() in this type of test
You can use the fixture or the UserFactory in most of the cases
look at the fixture to see what is really doing